joomla/joomla-cms
# [5.0][Events] Schemaorg event classes

### Summary of Changes

Implementing Event classes for Schemaorg events.

Affected events:

* [x] onSchemaBeforeCompileHead
* [x] onSchemaPrepareData
* [x] onSchemaPrepareForm
* [x] onSchemaPrepareSave

### Testing Instructions

- Apply patch (and delete autoload_psr4.php)
- Create schemas for articles and contacts in backend
- Check output in frontend

### Actual result BEFORE applying this Pull Request

works but support no concrete events

### Expected result AFTER applying this Pull Request

works with legacy and concrete events

### Link to documentations
Please select:
- [x] Documentation link for docs.joomla.org: [Plugin/Events#Schemaorg](https://docs.joomla.org/Special:MyLanguage/Plugin/Events#Schemaorg)
